includes/Sensor.h

```diff
@@ -106,16 +106,14 @@ class Barometer : public Sensor {
    public:
     Barometer(std::string name, Rocket& rocket, double refresh_rate,
               double noise_mean = 0.0f, double noise_stddev = 0.1f);
-    void update_data(double tStep);
-    double get_data();
+    void update_data(double tStep) override;
+    double get_data() override;
 
     void set_constant_bias(double bias) { bias_ = bias; };
 
    private:
     double data_;  // The sensor's current reading
 
-    double noise_;  // Noise value to be added to measurement
```

Comment:
Why did you delete this?

Reply:
I don't think there needs to me a variable to hold the noise when it is only ever used to add to the measured data. I've put it back for now though.